src/Evolution/Systems/Cce/AnalyticSolutions/RotatingSchwarzschild.cpp

```diff
@@ -0,0 +1,126 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/Cce/AnalyticSolutions/RotatingSchwarzschild.hpp"
+
+#include <complex>
+#include <vector>
+
+#include "DataStructures/ComplexDataVector.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/Tags.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Systems/Cce/Tags.hpp"
+#include "NumericalAlgorithms/Spectral/SwshCollocation.hpp"
+#include "NumericalAlgorithms/Spectral/SwshInterpolation.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Cce::Solutions {
+
+/// \cond
+RotatingSchwarzschild::RotatingSchwarzschild(const double extraction_radius,
+                                             const double mass,
+                                             const double frequency) noexcept
+    : SphericalMetricData{extraction_radius},
+      frequency_{frequency},
+      mass_{mass} {
+  // member variables of the parent class must be initialized in the body of the
```

Comment:
the constructor for `SphericalMetricData` should be initializing `extraction_radius_`